src/cli/test/task-generate.spec.ts

```diff
@@ -14,13 +14,15 @@ jest.mock('prompts', () => ({
 
 const setup = async () => {
   const sys = mockCompilerSystem();
-  const config: d.ValidatedConfig = mockValidatedConfig(sys);
-  config.configPath = '/testing-path';
-  config.srcDir = '/src';
+  const config: d.ValidatedConfig = mockValidatedConfig({
+    configPath: '/testing-path',
+    flags: { args: [], knownArgs: [], task: 'generate', unknownArgs: [] },
```

Comment:
same comment about `createConfigFlags` here

Reply:
D'oh - how did I forget we had that? Good catch!